src/hooks/useFuelWallet.ts

```
@@ -23,139 +25,123 @@ export const useFuelWallet = () => {
+      if (!fuel) {
+        setError('Fuel not initialized');
+        return;
+      }
+
+      try {
+        setIsConnecting(true);
+        setError(undefined);
+
+        // Get all available connectors
+        const connectors = await fuel.connectors();
+        
```

ESLint

[error] Delete `········`

(prettier/prettier)